worker/uniter/runner/context/context.go

```diff
@@ -1353,6 +1359,12 @@ func (ctx *HookContext) HookVars(
 	}
 	if ctx.workloadName != "" {
 		vars = append(vars, "JUJU_WORKLOAD_NAME="+ctx.workloadName)
+		if ctx.noticeType != "" {
+			vars = append(vars,
+				"JUJU_NOTICE_TYPE="+ctx.noticeType,
+				"JUJU_NOTICE_KEY="+ctx.noticeKey,
```

Comment:
JUJU_ or PEBBLE_ ??

Reply:
It's `JUJU_` because these are env vars for the Juju hook (used by ops). So they need to have the `JUJU_` prefix like all the others. We could do `JUJU_PEBBLE_NOTICE_TYPE`, but that seems unnecessary.